lib/Core/Site/Values/Content.php

```diff
@@ -429,4 +440,183 @@ private function getInnerOwnerUser()
 
         return $this->innerOwnerUser;
     }
+
+    public function getReverseFieldRelation(
```

Comment:
Isn't this supposed to use query executor from this PR, instead of building everything manually?